packages/sdk/client-services/src/packlets/invitations/invitations-manager.ts

```diff
@@ -239,6 +244,40 @@ export class InvitationsManager {
     return { ctx, stream, observableInvitation };
   }
 
+  private _createObservableAcceptingInvitation(handler: InvitationProtocol, initialState: Invitation) {
```

Comment:
extracted it from invitations-handler for API consistency with host-side invitation flow